addons/account/static/src/components/bills_upload/bills_upload.js

```diff
@@ -126,6 +127,14 @@ export const AccountMoveUploadKanbanView = {
 };
 
 // Accounting Dashboard
+export class DashboardKanbanDropdownMenuWrapper extends KanbanDropdownMenuWrapper {
+    onClick(ev) {
+        // Keep the dropdown open as we need the fileuploader to remain in the dom
+        if (![...ev.target.classList].includes("account_file_uploader") && !ev.target.closest('.file_upload_kanban_action_a')) {
```

Comment:
What about this?

```suggestion
        if (ev.target.tagName === "INPUT" && ![...ev.target.classList].includes("account_file_uploader")) {
```

Reply:
both those cases cover the same input element....
I have adapted for the `input` and the `span`